.github/workflows/codespell.yml

```diff
@@ -0,0 +1,25 @@
+# Codespell configuration is within setup.cfg
```

Comment:
since pre-commit kicks in on PRs, I could drop this workflow -- you would just loose annotation of lines where typos were introduced by the `codespell-project/codespell-problem-matcher@v1`